Amazon.QLDB.Driver.Serialization/Amazon.QLDB.Driver.Serialization.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AWSSDK.QLDBSession" Version="3.5.2" /> 
+    <ProjectReference Include="..\Amazon.QLDB.Driver\Amazon.QLDB.Driver.csproj" />
+    <ProjectReference Include="..\..\..\alpian\ion-object-mapper-dotnet\Amazon.Ion.ObjectMapper\Amazon.Ion.ObjectMapper.csproj" />

```

Comment:
Since the object mapper isn't released yet, I think we can use [git submodules](https://gist.github.com/gitaarik/8735255) within this repo and then get the build passing again

Reply:
This can't be merged until the object mapper is a public thing, you are correct.